docs/types.md

```diff
@@ -782,7 +782,7 @@ All types of FileSpecs share some common fields:
 
 ### FileSpec -- Path
 
-A FileSpec with `type` equal to `path` points to an absolute or relative path accessible from the current machine.
+A FileSpec with `type` equal to `path` points to an absolute or relative path accessible from the current machine. The absolute or relative path may also include a glob pattern to target multiple files or directories.
```

Comment:
s/absolute or relative path/path/

KISS
Documentation follows the same rule as code.